ui/src/pipeline/pipeline.module.ts

```diff
@@ -18,12 +23,33 @@ export default class PipelineModule extends VuexModule {
     PIPELINE_SERVICE_URL,
   );
 
+  get pipelinesByDatasourceId(): (datasourceId: number) => Pipeline[] {
+    return (datasourceId: number): Pipeline[] => {
```

Comment:
Ok thanks.
I will remove it, because I think that was something for the future (I am not 100% myself anymore).
If we need it at some point we can just add it again (or the other suggestion)